src/main/java/racingcar/domain/Cars.java

```diff
@@ -33,6 +33,15 @@ public List<Car> getValues() {
     return values;
   }
 
+  public List<Car> findWinners() {
+    Car winnerCar = values.stream()
+        .max(Car::compareTo)
+        .orElseThrow(IllegalStateException::new);
```

Comment:
```suggestion
    Car winner = Collections.max(values);
```
Comparable를 구현한걸 활용해서 위처럼 할 수도 있겠네요 :slightly_smiling_face: